search/tests/mock_search_engine.py

```diff
@@ -47,6 +48,11 @@ def _find_field(doc, field_name):
         return field_value
 
 
+def _is_iterable(item):
```

Comment:
@Kelketek the warning is accurate. This method is later used to check if document field value is iterable, and search value is not. The second part of warning (searching lists in multivalue fields) does not apply here.

However, basically this MockSearchEngine should replicate elasticsearch behavior as closely as possible (at least for now we decided that it will), and elastic does not document what would happen if array is passed to `term` filter: http://www.elastic.co/guide/en/elasticsearch/reference/0.90/query-dsl-term-filter.html

Also, note elasticsearch 0.90 is used - I haven't seen any provision scripts that would confirm that, but live devstack instanceon my machine uses 0.90 for sure.